jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -325,9 +254,87 @@ public boolean isContainedIn(Resource r)
     }
 
     @Override
-    public URI getAlias()
+    public URI getTargetURI()
     {
-        return this.alias == null ? null : this.alias.toUri();
+        if (!targetResolved)
+        {
+            alias = resolveTargetAliasPath();
+            targetResolved = true;
+        }
+        if (alias == null)
+            return null;
+        return alias.toUri();
+    }
+
+    private Path resolveTargetAliasPath()
```

Comment:
Should be renamed to just `resolveTargetPath()` as _this_ `Resource` is the alias.